.circleci/config.yml

```diff
@@ -48,7 +51,7 @@ workflows:
           requires:
             - orb-tools/pack
             - shellcheck/check
-          vcs-type: github
+          vcs-type: << pipeline.project.type >>
```

Comment:
Why this change?

Reply:
`pipeline.project.type` is always defined in Circle, I think it's cleaner to use envvar than have it hardcoded